From: Jiri Slaby <jslaby@suse.cz>
Date: Mon, 18 Jan 2016 18:00:39 +0100
Subject: kgr: mark some more missed kthreads
Patch-mainline: not yet, kgraft
References: bnc#962336

When searching for unhandled kthreads in SLE12-SP2 I found out these
in SP0+1.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/isdn/mISDN/l1oip_core.c             |    1 +
 drivers/staging/lustre/lnet/lnet/acceptor.c |    1 +
 fs/btrfs/inode-map.c                        |    1 +
 3 files changed, 3 insertions(+)

--- a/drivers/isdn/mISDN/l1oip_core.c
+++ b/drivers/isdn/mISDN/l1oip_core.c
@@ -740,6 +740,7 @@ l1oip_socket_thread(void *data)
 				printk(KERN_WARNING
 				       "%s: broken pipe on socket\n", __func__);
 		}
+		kgr_task_safe(current); /* insufficient */
 	}
 
 	/* get socket back, check first if in use, maybe by send function */
--- a/drivers/staging/lustre/lnet/lnet/acceptor.c
+++ b/drivers/staging/lustre/lnet/lnet/acceptor.c
@@ -387,6 +387,7 @@ lnet_acceptor(void *arg)
 		return rc;
 
 	while (!lnet_acceptor_state.pta_shutdown) {
+		kgr_task_safe(current); /* insufficient */
 
 		rc = libcfs_sock_accept(&newsock, lnet_acceptor_state.pta_sock);
 		if (rc != 0) {
--- a/fs/btrfs/inode-map.c
+++ b/fs/btrfs/inode-map.c
@@ -54,6 +54,7 @@ static int caching_kthread(void *data)
 	key.offset = 0;
 	key.type = BTRFS_INODE_ITEM_KEY;
 again:
+	kgr_task_safe(current);
 	/* need to make sure the commit_root doesn't disappear */
 	down_read(&fs_info->commit_root_sem);
 
